lib/index.js

```diff
@@ -1,33 +1,58 @@
 const standard = require('standard');
+const yaml = require('js-yaml');
 
 module.exports = {
-  async push(event, context) {
-    const push = event.payload;
+  async push(context) {
+    let config, whiteList;
+    let linterItems = new Object();
+    const push = context.payload;
 
     const compare = await context.github.repos.compareCommits(context.repo({
       base: push.before,
       head: push.after
     }));
 
     const branch = push.ref.replace('refs/heads/', '');
+    //const branch = 'refs/heads/revert-28-hiimbex-patch-11';
+    // Checks for a config file
+    try {
+      const options = context.repo({path: '.github/linter.yml'});
+      const response = await context.github.repos.getContent(options);
+      config = yaml.safeLoad(Buffer.from(response.data.content, 'base64').toString()) || {};
+    } catch (err) {
+      if (err.code !== 404) throw err;
+    }
+    // Adds properties to a LinterItem object to be passed to standard.lintText()
+    if (config) for (property in config) {
```

Comment:
Haha probably not, but I personally think it looks cleaner. (I didn't actually lint this 😬 ) But I'll update it when I lint things!